packages/util/test/freeable.test.ts

```diff
@@ -81,6 +81,7 @@ describe('freeable', () => {
         resolve(returnValue);
         await expect(result).resolves.toBe(returnValue);
         expect(freeableSpy).toBeCalledTimes(1);
+        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
         reject;
```

Comment:
This doesn't seem to be doing anything, maybe just delete it?

Reply:
I was unsure wether it does something or not 😄

Reply:
Updated in [a56da0f](https://github.com/input-output-hk/cardano-js-sdk/pull/584/commits/a56da0ffc0122fb0e48d6478f3f811d0d8b7e89d)